### PR TITLE
9429 append clinic letter to receipt

### PR DESCRIPTION
**Pull request by TomElliottFlexion**

### CODE REVIEW

File: shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.js [hunk 1]
```diff
@@ -162,10 +168,39 @@ const generateNoticeOfReceipt = async ({ applicationContext, caseEntity }) => {
...
+
+  const doesClinicLetterExist = await applicationContext
+    .getPersistenceGateway()
+    .isFileExists({
+      applicationContext,
+      key: clinicLetterKey,
+    });
+
+  if (doesClinicLetterExist) {
+    const clincLetter = await applicationContext
```
Comment (rosiehsmith) [inline shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.js]: nit: spelling!

**Review by rosiehsmith — APPROVED** [review]
Nice! Great tests.

File: shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.js (hunk 1 above)
Comment (TomElliottFlexion) [reply shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.js]: Nice catch, thanks!

File: shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.js [hunk 2]
```diff
@@ -162,10 +168,39 @@ const generateNoticeOfReceipt = async ({ applicationContext, caseEntity }) => {
...
+      key: clinicLetterKey,
+    });
+
+  if (doesClinicLetterExist) {
+    const clinicLetter = await applicationContext
+      .getPersistenceGateway()
+      .getDocument({
+        applicationContext,
+        key: clinicLetterKey,
+        protocol: 'S3',
```
Comment (rachelschneiderman) [inline shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.js:189]: Possible DOD, extract all of these s3 strings to an entity constant?

File: web-client/integration-tests/journey/docketClerkVerifiesPetitionReceiptLength.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import { PDFDocument } from 'pdf-lib';
```
Comment (rachelschneiderman) [inline web-client/integration-tests/journey/docketClerkVerifiesPetitionReceiptLength.js:1]: Nit: can this import be standardized like the one below?

**Review by rachelschneiderman — APPROVED** [review]

File: shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.js (hunk 2 above)
Comment (TomElliottFlexion) [reply shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.js:189]: Ooh that is a good idea

**Review by mrinsin — APPROVED** [review]

**Pull request merged**